<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="contrastAdjust">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>21</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightnessAdjust">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>51</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>241</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getHistogram">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalizeHistogram">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MatchHistogram">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>450</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Match</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate90">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate 90Â°</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyFilter">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>420</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Convolution</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>General</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="imageList">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="filterList">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Images</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addImage_action_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFilter_action_button">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetImage">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFilter">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="matchHistogramFIeld">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addFilter_text_field">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addImage_file_path">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>170</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>in</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>400</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>filter</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="rotate90Field">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="brightnessField">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zoomInField">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zoomOutXFIeld">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>x factor</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="zoomOutYFIeld">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>270</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>y factor</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="getGrayscale">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Grayscale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getNegative">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verticalMirror">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>V-mirror</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HorizontalMirror">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>H-mirror</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quantize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outImageField">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>out</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantizationField">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="contrastField">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteImage">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pre_clamp_offset">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
